modules.json

```diff
@@ -17,11 +17,15 @@
                         "branch": "master",
                         "git_sha": "c8e35eb2055c099720a75538d1b8adb3fb5a464c"
                     },
-                    "minimap2/align": {
+                    "minimap2/index": {
                         "branch": "master",
                         "git_sha": "c8e35eb2055c099720a75538d1b8adb3fb5a464c"
                     },
-                    "minimap2/index": {
```

Comment:
were these changes managed by nf-core tools module update/install?

Reply:
perhaps? I never manually update this file